agimus_controller_ros/agimus_controller_ros/simple_trajectory_publisher.py

```diff
@@ -146,10 +154,10 @@ def publish_mpc_input(self):
         # Create the message
         msg = MpcInput()
         msg.w_q = [1.0] * self.croco_nq
-        msg.w_qdot = [1e-2] * self.croco_nq
+        msg.w_qdot = [1e-1] * self.croco_nq
         msg.w_qddot = [1e-6] * self.croco_nq
-        msg.w_robot_effort = [1e-4] * self.croco_nq
-        msg.w_pose = [1e-10] * 6
+        msg.w_robot_effort = [3e-4] * self.croco_nq
+        msg.w_pose = [1e-1] * 6
```

Comment:
I think I have seen a change in those weights at each PR modifying this file.

Could you add ROS node parameters to set those values?